dotenv/Cargo.toml

```diff
@@ -0,0 +1,47 @@
+# THIS FILE IS AUTOMATICALLY GENERATED BY CARGO
```

Comment:
You can remove those lines, they're not really useful as they were added by crates.io. They're only here because I left them in when I imported the package from there